### ISSUE TITLE
Has anyone else been spammed by "K2 Partnering Solutions" ...?

### ISSUE THREAD

**Issue by nelsonic** [drcongo/spammy-recruiters#46]
![k2-partnering-solutions-spam](https://cloud.githubusercontent.com/assets/194400/14003472/e0e213a6-f14a-11e5-850c-7da267d61d65.png)

**Comment by drcongo** [comment]
Nope, but funnily enough, I started this repo recently: https://github.com/drcongo/spammy-offshorers
